drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationUtils.java

```diff
@@ -232,6 +233,17 @@ public static ScenarioGridColumn getScenarioGridColumn(ScenarioSimulationBuilder
         return headerBuilder;
     }
 
+    protected static double getColumnWidth(String columnId) {
+        switch (columnId) {
+            case "Index":
```

Comment:
You can use ExpressionIdentifier.INDEX.getName()
WDYT?

Reply:
I had to create an enum, but I think it was somehow missing anyway, so better that way. It is a little bit more verbose but less "magic words" around